stdlib/internals.ncl

```diff
@@ -41,54 +41,45 @@
     contract (%insert_type_variable% sealing_key polarity label) value,
 
   "$enums" = fun case label value =>
-      if %typeof% value == `Enum then
-          %assume% case label value
-      else
-          %blame% (%label_with_message% "not an enum tag" label),
+    if %typeof% value == `Enum then
+      %assume% case label value
+    else
+      %blame% (%label_with_message% "not an enum tag" label),
 
   "$enum_fail" = fun label =>
-      %blame% (%label_with_message% "tag not included in the enum type" label),
+    %blame% (%label_with_message% "tag not included in the enum type" label),
 
   "$record" = fun field_contracts tail_contract label value =>
     if %typeof% value == `Record then
       # Returns the sub-record of `l` containing only those fields which are not
       # present in `r`. If `l` has a sealed polymorphic tail then it will be
       # preserved.
-      let field_diff = fun left right => array.fold_left
-        (fun acc field =>
-          if %has_field% field right then
-            acc
-          else
-            %record_insert% field acc (left."%{field}"))
-        (%record_empty_with_tail% left)
-        (%fields% left)
+      let field_diff =
+        fun left right =>
+          array.fold_left (fun acc field =>
+            if %has_field% field right then
+              acc
+            else
+              %record_insert% field acc (left. "%{field}")) (%record_empty_with_tail% left) (%fields% left)
       in
-
-      let contracts_not_in_value = field_diff field_contracts value in
-      let missing_fields = %fields% contracts_not_in_value in
-
-      if %length% missing_fields == 0 then
-        let tail_fields = field_diff value field_contracts in
-        let fields_with_contracts = array.fold_left
-          (fun acc field =>
-            if %has_field% field field_contracts then
-              let contract = field_contracts."%{field}" in
-              let label = %go_field% field label in
-              let val = value."%{field}" in
-              %record_insert% field acc (%assume% contract label val)
+        let contracts_not_in_value = field_diff field_contracts value in
+          let missing_fields = %fields% contracts_not_in_value in
+            if %length% missing_fields == 0 then
+              let tail_fields = field_diff value field_contracts in
+                let fields_with_contracts =
+                  array.fold_left (fun acc field =>
+                    if %has_field% field field_contracts then
+                      let contract = field_contracts. "%{field}" in
+                        let label = %go_field% field label in
+                          let val = value. "%{field}" in
+                            %record_insert% field acc (%assume% contract label val)
```

Comment:
Here is a very nice diagonal that demonstrates that we don't want to indent the body of a let-binding :sweat_smile: